schema/schema.sql

```diff
@@ -86,13 +86,13 @@ CREATE TABLE gutptag (
 );
 
 CREATE TABLE gutptag_idhash (
-  id varchar primary key,		
+  id varchar primary key,
   hash varchar not null
 );
 
 -- The M:N index mapping table
 CREATE TABLE gutpposttag (
-  id varchar primary key,		-- the post id
+  post_id varchar primary key,		-- the post id
```

Comment:
for complying with the `EightFishModel` trait, we'd better enforce the name of the id field is `id`

Reply:
ah, you are right about the id limitation in EightFishModel.
but it just make it hard to read or use the schema if we end up with using keys from other table.
and in this case, the key we actually need should be a compound key (post_id, tag_id), 
that also means the id should not be primary key, it will very likely none unique.

maybe we should think to support alias or something in EightFish that support other field as primary key.

Reply:
We could remove primary key constraints at first. And keep id field currently.